gammapy/data/event_list.py

```diff
@@ -393,6 +393,33 @@ def peek(self):
         plt.tight_layout()
         plt.show()
 
+    def plot_image(self, ax= None, number_bins=10):
+        """Plot the counts as a function of x and y camera coordinate."""
+        import matplotlib.pyplot as plt
+        ax = plt.gca() if ax is None else ax
+
+        max_x = max((self[:]['COREX']))
```

Comment:
COREX / COREY is the reco event core position on the ground.
Use DETX / DETY here, which are the "field of view coordinates"
(I'll write up a spec what exactly those are soon [here](http://gamma-astro-data-formats.readthedocs.org/en/latest/info/coordinates.html#field-of-view)).